From 27b924ad88b53c1bff736d144b90ce655087d1a5 Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Fri, 29 Oct 2021 11:19:10 +0200
Subject: [PATCH 1/3] shortcut skip-unbootable-disks optimitation

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
RH-MergeRequest: 6: shortcut skip-unbootable-disks optimitation
RH-Commit: [1/1] 95008c119b45b4a360caa4a7733420d72aec99cb
RH-Bugzilla: 2073012
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Oliver Steffen <osteffen@redhat.com>
RH-Acked-by: Pawel Polawski <ppolawsk@redhat.com>
---
 src/boot.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/boot.c b/src/boot.c
index 1effd802..b18e7788 100644
--- a/src/boot.c
+++ b/src/boot.c
@@ -297,11 +297,16 @@ find_prio(const char *glob)
 
 u8 is_bootprio_strict(void)
 {
+#if 0
     static int prio_halt = -2;
 
     if (prio_halt == -2)
         prio_halt = find_prio("HALT");
     return prio_halt >= 0;
+#else
+    // force initializing all disks
+    return 0;
+#endif
 }
 
 int bootprio_find_pci_device(struct pci_device *pci)
-- 
2.35.3

